src/index.html

```diff
@@ -10,7 +12,35 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
+      action="#"
+      method="get"
+      data-qa="big">
+      <label>
+        <input
+          class="bar theBigOne"
+          type="search"
+          placeholder="Try &quot;Los Angeles&quot"
+          data-qa="keypress"
+        >
+      </label>
+    </form>
+
+    <form
+      action="#"
+      method="get"
+      data-qa="small">
```

Comment:
не забувай перености закриваючий тег
```suggestion
      data-qa="small"
    >
```